src/olympia/discovery/tests/test_admin.py

```diff
@@ -19,6 +19,22 @@ class TestDiscoveryAdmin(TestCase):
     def setUp(self):
         self.list_url = reverse('admin:discovery_discoveryitem_changelist')
 
+    def _get_heroform(self, item_id):
```

Comment:
The tests in this file got removed (moved to https://github.com/mozilla/addons-server/blob/master/src/olympia/promoted/tests/test_admin.py but I think I already made the relevant changes when I moved them) so I'm not sure there's anything left you need to change in discover/tests/test_admin.py

Reply:
It has now been removed in the latest push.